index.bs

```diff
@@ -3689,29 +3692,27 @@ partial interface MLGraphBuilder {
     The behavior of this operation when the axes parameter is set to [1,2,3] can be [EMULATED]
   </summary>
   <pre highlight="js">
-    // The reduction of the mean and variance values happens over the spatial dimensions 
-    // across all the input features (i.e. all channels) of the input tensor.
-    const reduceOptions = { axes: [1,2,3], keepDimensions: true };
+    // The reduction of the mean and variance values happens over the spatial
+    // dimensions across all the input features (i.e. all channels) of the input
+    // tensor.
+    const reduceOptions = {
+      axes: [1, 2, 3],
+      keepDimensions: true
+    };
```

Comment:
```suggestion
    const reduceOptions = {
        axes: [1, 2, 3],
        keepDimensions: true
    };
```
(nit) If we're settling on 4 column indents elsewhere in this snippet, then I'd apply it consistently (as mixing 2-column and 4-column indents haphazardly is harder to follow and looks wonky).

Reply:
Future PR will apply clang-format everywhere.

Reply:
> Future PR will apply clang-format everywhere.

`<ignorable>`Ah yes, the formatting bulldozers. So there's *conformity* and then there's the true underlying goal of *readability*, where conformity often *aids* readability, but when applied blindly often *hampers* it too. I found myself needing `clang-format off` so often that I gave up on it for my own projects 😅, but it's becoming more [configurable over time](https://stackoverflow.com/questions/50689027/get-clang-format-to-put-closing-parenthesis-of-multiline-function-calls-on-separ/53935614?noredirect=1#comment125999235_53935614)... I'm still looking for a really good one, but I may have to wait for one implemented as a trained neural network 😉, because the interaction between all the rules are quite complicated.`<ignorable>`